README.md

```diff
@@ -22,6 +25,10 @@ Feel free to submit a pull request with your suggestions of awesome github profi
 
 **Note** : Please do not edit anything below `<!--START_SECTION:data-section-->` in `README.MD` our [GitHub Action](https://github.com/EddieJaoudeCommunity/gh-action-community) will make the necessary changes for you.
 
+4. Create a PullRequest to add the changes to this Ropository. Please change the title to `add your-username` and in the description add `resolve #issueNumber`
```

Comment:
I think Pull Request should be two words. Looks like there is a typo in Repository. :nerd_face: